src/main/java/duke/Task.java

```diff
@@ -0,0 +1,59 @@
+package duke;
+
+/**
+ * Represents a Task that can be stored by Duke
+ */
+public abstract class Task {
```

Comment:
Is there a need to make Task an abstract class? It has many default implementations of various methods, and perhaps in future we want to allow the user to add a generic task to Duke.